develop/README.md

```diff
@@ -0,0 +1,32 @@
+# Docker
+
+## Installing ONLYOFFICE Docs
+
+## How to use - Linux
+
+**Note**: You need the latest Docker version installed.
+
+You might need to pull **onlyoffice/documentserver** image:
+
+```bash
+sudo docker pull onlyoffice/documentserver
+```
+
+### Create develop image
+To create a image with the ability to include external non-minified sdkjs code, use the following command from the dockerfile directory:
+```bash
```

Comment:
There is `markdownlint` check on `release/v6.1.0` branch https://github.com/ONLYOFFICE/build_tools/blob/release/v6.1.0/.github/workflows/check.yml#L15

I've run it on this file also, to have clear style of markdown files, like newlines and etc...